src/firefly/js/drawingLayers/Catalog.js

```diff
@@ -52,10 +52,11 @@ var createCnt= 0;
 function creator(initPayload, presetDefaults) {
     const {catalogId, tableData, tableMeta, title,
            selectInfo, columns, tableRequest, highlightedRow, color, angleInRadian=false,
+           symbol, size,
            dataTooBigForSelection=false, catalog=true,boxData=false }= initPayload;
     var drawingDef= makeDrawingDef();
-    drawingDef.size= 5;
-    drawingDef.symbol= DrawSymbol.SQUARE;
+    drawingDef.size= size || 5;
+    drawingDef.symbol= DrawSymbol.get(symbol) || DrawSymbol.SQUARE;
     drawingDef= Object.assign(drawingDef,presetDefaults);
```

Comment:
side default has not changed, it just now is allowed to be passed as a parameter to the drawing layer. The default size is `5`.